code/modules/projectiles/gun.dm

```diff
@@ -193,7 +200,11 @@
 	if(flag)
 		if(user.zone_selected == "mouth")
 			if(target == user && HAS_TRAIT(user, TRAIT_BADASS)) // Check if we are blowing smoke off of our own gun, otherwise we are trying to execute someone
-				user.visible_message("<span class='danger'>[user] blows smoke off of [src]'s barrel. What a badass.</span>")
+				user.visible_message(
+					"<span class='danger'>[user] blows smoke off of [src]'s barrel. What a badass.</span>",
+					"<span class='danger'>You blow smoke off of [src]'s barrel.</span>",
+					"<span class='danger'>You hear someone blowing.</span>"
```

Comment:
this is maybe a bit too vague and non-descriptive even for a blind message

Reply:
Updated.